packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -36,6 +36,28 @@ const Color _kLightThemeSplashColor = const Color(0x66C8C8C8);
 const Color _kDarkThemeHighlightColor = const Color(0x40CCCCCC);
 const Color _kDarkThemeSplashColor = const Color(0x40CCCCCC);
 
+/// Configures the tap target and layout size of certain Material widgets.
+///
+/// The following widgets are impacted:
+///
+///   * [FloatingActionButton], only the mini tap target size is increased.
+///   * [MaterialButton]
+///   * [OutlineButton]
+///   * [FlatButton]
+///   * [TimePicker]
+///   * [SnackBar]
+enum MaterialTapTargetSize {
+  /// Expands the minimum tap target size to 48dp by 48dp.
+  ///
+  /// This is the default value of [ThemeData.materialHitTestSize] and the
+  /// recommended behavior to conform to Android accessibility recommendations.
+  expanded,
+
+  /// Collapses the tap target size to the minimum provided by the Material
+  /// specification.
+  collapsed,
+}
```

Comment:
"expanded" sounds like it's bigger than usual. Maybe "padded" and "shrinkWrap"?

Reply:
Updated

Reply:
Fixed